allensdk/internal/api/mtrain_api.py

```diff
@@ -0,0 +1,26 @@
+import psycopg2
+import pandas as pd
+
+from . import PostgresQueryMixin, one
+
+class MtrainApi(PostgresQueryMixin):
+
+    def __init__(self, dbname="mtrain", user="mtrainreader", host="prodmtrain1", password="mtrainro", port=5432):
```

Comment:
Do you need this constructor?  Looks like it's just a passthrough.

Reply:
This sets the database access settings via the call to the parent constructor.  If I dont set them here, it will try and connect to LIMS.